test/server.test.js

```diff
@@ -2051,6 +2051,34 @@ test(
     }
 );
 
+// eslint-disable-next-line max-len
+test("should emit 'after' on client closed request with an uncaughtException", function(t) {
+    SERVER.on('after', function(req, res, route, err) {
+        t.ok(err);
+        t.equal(req.connectionState(), 'close');
+        t.equal(res.statusCode, 444);
+        t.equal(err.name, 'Error');
+        t.end();
+    });
+
+    SERVER.on('uncaughtException', function(req, res, route, err, next) {
```

Comment:
In the documentation, I would suggest not calling that callback `next`, so that users don't confuse this callback with middlewares' next callbacks and their specific semantics/API.